server/game/core/gameSteps/ActionWindow.js

```diff
@@ -32,13 +32,12 @@ class ActionWindow extends UiPrompt {
         }
 
         // IMPORTANT: the below code is referenced in the debugging guide (docs/debugging-guide.md). If you make changes here, make sure to update that document as well.
```

Comment:
Let's move this comment down to `getCardLegalActions` and we can update the wiki to refer there instead, since that's where the important values are being computed now

Reply:
yessir!